torch/csrc/jit/passes/utils/subgraph_utils.h

```diff
@@ -23,11 +23,20 @@ TORCH_API Node* createSingletonSubgraph(Node* n, Symbol subgraphKind);
 // Merge a node into a subgraph node. If `toMerge` is also a subgraph, the
```

Comment:
Do we not need to update vmap in this case? https://github.com/pytorch/pytorch/pull/43235/files

Reply:
Or in this case ? https://github.com/pytorch/pytorch/pull/43235/files#diff-2589d5e7534a9f39ac403b29ff7bb166R23

Reply:
I'm not sure where these links are pointing to? Is the question about `createSingletonSubgraph` function?

Reply:
The only user of merging nodes with blocks afaik is the functional graph stuff which isn’t really being used or maintained, so we could consider removing this behavior

Reply:
Ah, I see your point. Let me add a TODO so that at least it doesn't get unnoticed - in the long term I agree that it might be worth just removing this functionality if we don't really use it.